metrics.md

```diff
@@ -30,6 +30,26 @@ The following are the minimal metrics agreed to be conformed by the various clie
 | `beacon_processed_deposits_total`          | Gauge       | Total number of deposits processed                          | On epoch transition  |
 
 \* All `*_root` values are converted to signed 64-bit integers utilizing the last 8 bytes interpreted as little-endian (`int.from_bytes(root[24:32], byteorder='little', signed=True)`).
+
+### PeerDAS Metrics
+
+The following metrics are proposed to be added to clients for PeerDAS monitoring. This list is open for discussion. Each client has the opportunity to contribute to it by suggesting additions or disputing existing metrics.
+
+#### Data column, kzg, custody metrics
+
+| Name | Metric type | Usage | Sample collection event |
+|--------------------------------------------|-------------|-------------------------------------------------------------|----------------------|
+| `beacon_data_column_sidecar_processing_requests_total`            | Counter   | Number of data column sidecars submitted for processing                     | On data column sidecar gossip verification     |
+| `beacon_data_column_sidecar_processing_successes_total`           | Counter   | Number of data column sidecars verified for gossip                         | On data column sidecar gossip verification     |
+| `beacon_data_column_sidecar_gossip_verification_seconds`          | Histogram | Full runtime of data column sidecars gossip verification                   | On data column sidecar gossip verification     |
+| `beacon_data_availability_reconstructed_columns_total`            | Counter   | Total count of reconstructed columns                                      | On data column kzg verification  |
+| `beacon_data_availability_reconstruction_time_seconds`            | Histogram | Time taken to reconstruct columns                                      | On data column kzg verification  |
```

Comment:
A few more reconstruction metrics that could be useful:

- `kzg_data_column_reconstruction_attempts`: Count of times data column reconstruction has been attempted
- `kzg_data_column_reconstruction_failures`: Count of times data column reconstruction has failed

Lighthouse source for this [here](https://github.com/sigp/lighthouse/blob/e31ac508d404700c35d99936028a5fd74749c335/beacon_node/beacon_chain/src/metrics.rs#L1890-L1904)